<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoanWidget</class>
 <widget class="QWidget" name="LoanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="mainLayout" stretch="2,8">
     <item>
      <layout class="QVBoxLayout" name="titleLayout">
       <item>
        <layout class="QHBoxLayout" name="titleBar">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font-size: 30px;
color: white;</string>
           </property>
           <property name="text">
            <string>Loans Information</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="styleSheet">
            <string notr="true">QPushButton{
background-color: #274c77;
color:#FFFAFB;
font:bold;
padding: 5px;
border-radius:7px;
width: 100px;
margin-right:15px;
height: 25px;
}

QPushButton:hover {
  background-color: #4cc9f0;
  color:black;
font:bold;
padding: 5px;
border-radius:7px;
}

QPushButton:clicked{
 background-color: rgb(255, 233, 67);
  color:black;
font:bold;
padding: 3px;
border-radius:10px;

}</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="filterBar">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="pendingCheckbox">
           <property name="styleSheet">
            <string notr="true">color:white
</string>
           </property>
           <property name="text">
            <string>Pending</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="orgField">
           <property name="styleSheet">
            <string notr="true">color: white;
font-size: 16px
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="tableLayout">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
